### DISCUSSION TITLE
Airflow on Openshift and remote Celery Worker

### DISCUSSION THREAD

**Discussion by MattiaGallegati** [apache/airflow#24407] [Q&A]
Hello everyone,
As part of my Airflow adoption I'm will try to set up Airflow running on a Openshift cluster, using the helm chart provided and Celery executor. 
Then I would like to connect my installation to a Celery worker which is on a single machine outside the cluster.
Is Openshift considered "supported" just as Kubernetes?
Can I connect my installation to a "remote" worker installed on a machine outside the cluster?

**Comment by potiuk** [comment]
Yes it should be, there is no "specific" things that would prevent it, but you will get "guarantees" . If it works, cool. if it does not - you still have the "right to repair" and fix any issues and contribute back.

**Reply by khalsz** [reply]
Hi, I am having the same problem. I am running my airflow dag on my local pc, then I have a standalone celery worker on a remote pc. I want the celery worker on the remote pc to execute the task. so I make sure the celery worker is listening to the same broker the airflow celery executor on my local pc is sending message to. I achieved this via IP. WHile the celery worker connects to the broker, it is not listening to the queued messaged on the broker and the task is not executing. I have been battling this for some time. Any help please?

**Reply by khalsz** [reply]
Although, I am running a celery worker remotely to execute the task, I am a bit confused if the right thing would be to configure the executor as a custom executor or celery executor since the celery executor is not running inside airflow.
